components/mpas-ocean/src/analysis_members/mpas_ocn_mixed_layer_depths.F

```diff
@@ -286,11 +286,11 @@ subroutine ocn_compute_mixed_layer_depths(domain, timeLevel, err)!{{{
                !$omp end do
                !$omp end parallel
 
-               if (associated(landIceMask) ) then
+               if (associated(landIceFloatingMask) ) then
                   !$omp parallel
                   !$omp do schedule(runtime)
                   do iCell = 1, nCells
-                     tThreshMLD(iCell) = tThreshMLD(iCell) - abs(landIceDraft(iCell))*landIceMask(iCell)
+                     tThreshMLD(iCell) = tThreshMLD(iCell) - abs(landIceDraft(iCell))*landIceFloatingMask(iCell)
```

Comment:
This code is weird before your change.  I think maybe we should not multiply the `landIceDraft` by anything at all because I think we want to exclude the `landIceDraft` from the mixed layer depth even when we're in the open ocean it's being depressed by `landIcePressure` (the smoothing we discussed today).

I would change the `associated` above here to check for `landIceDraft` instead and remove the multiplication by `landIceMask` or `landIceFloatingMask`.